src/librbd/ImageWatcher.cc

```diff
@@ -163,42 +163,45 @@ void ImageWatcher<I>::notify_snap_create(const std::string &snap_name,
 template <typename I>
 void ImageWatcher<I>::notify_snap_rename(const snapid_t &src_snap_id,
 				         const std::string &dst_snap_name,
-                                      Context *on_finish) {
+					 Context *on_finish) {
   assert(m_image_ctx.owner_lock.is_locked());
   assert(m_image_ctx.exclusive_lock &&
          !m_image_ctx.exclusive_lock->is_lock_owner());
 
-  notify_lock_owner(SnapRenamePayload(src_snap_id, dst_snap_name), on_finish);
+  notify_lock_owner(SnapRenamePayload(src_snap_id, dst_snap_name, cls::rbd::UserSnapshotNamespace()), on_finish);
 }
 
 template <typename I>
 void ImageWatcher<I>::notify_snap_remove(const std::string &snap_name,
+					 const cls::rbd::SnapshotNamespace &snap_namespace,
                                          Context *on_finish) {
```

Comment:
@VictorDenisov Per the coding standard, it should be on the same line -- but code added before the coding standard usually have the bracket on the next line.